packages/restapi/src/lib/spaceV2/types/index.ts

```diff
@@ -0,0 +1,13 @@
+import { SPACE_REQUEST_TYPE } from "../../payloads";
+
+export type SpaceInviteInputOptions = {
```

Comment:
can convert to an Interface and keep in the same file instead of maintaining a separate types file, since most functions will have independent/unique types and making a types.ts file for common types (which will be very few, maybe ~3) doesnt help much.